### ISSUE TITLE
Failure at tester().enterText("", intoViewWithAccessibilityLabel: "")

### ISSUE THREAD

**Issue by waltermvp** [kif-framework/KIF#841]
Test always fails when `tester().enterText("", intoViewWithAccessibilityLabel: "")` is called. The exception if the following :

`Failure in child step: The step timed out after 1.00 seconds: Failed to get text "walter.vargaspena@gmail.com" in field; instead, it was "(null)"`

Some possible important info:
- It's an iPad app and the keyboard may take too long to present. 
- I do see the textField get populated with the text but it still fails.
- First time using KIF so it may be that im using the wrong fuctions?

Thanks in advance.

**Comment by phatmann** [comment]
Might be related to #649. 

Any chance you could dive in a little deeper to see what is going on?
